extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/hll/HllSketchMergeAggregatorFactory.java

```diff
@@ -142,6 +151,22 @@ public VectorAggregator factorizeVector(VectorColumnSelectorFactory selectorFact
     );
   }
 
+  private void validateInputs(@Nullable ColumnCapabilities capabilities)
+  {
+    if (capabilities != null) {
+      final ColumnType type = capabilities.toColumnType();
+      if (!ColumnType.UNKNOWN_COMPLEX.equals(type) && !TYPE.equals(type) &&
+          !(ValueType.COMPLEX.equals(type.getType()) &&
+            (Objects.equals(type.getComplexTypeName(), "HLLSketch") ||
+             Objects.equals(type.getComplexTypeName(), "HLLSketchBuild")))) {
+        throw new UOE("Using aggregation type %s is not supported for %s column. "
+                      + "Use a different aggregator type and run the query again.",
+                      getIntermediateType().getComplexTypeName(),
+                      type);
```

Comment:
Incorrect formatting

Reply:
I'm rephrasing this message as per the other review comment. But why was this incorrect formatting? Since we don't have a formatter integrated, I mostly try to keep the line-lengths restricted, what else should I keep in mind?

Reply:
Its like:
```java
     throw new UOE(a,
                               b);
```

Correct would be something like: 
```suggestion
        throw new UOE(
        "Using aggregation type %s is not supported for %s column. "
                      + "Use a different aggregator type and run the query again.",
                      getIntermediateType().getComplexTypeName(),
                      type
         );
```

The code style doesn't catch such errors afair.

Reply:
@LakshSingla 
The suggestion seems like
```
     throw new UOE(
      a,
               b);
```
which feels weird? 😅

Reply:
It's the positioning of the first string column and the last `);` in your code which is inconsistent.